src/views/DepositWithdraw/components/WithdrawInfo/index.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { colors, GU } from 'components/theme';
+
+import { ReactComponent as EthereumSvg } from 'assets/icons/ethereum.svg';
+import { ReactComponent as FarmSvg } from 'assets/icons/farm.svg';
+
+import {
+  MediumInfoCard,
+  SmallInfoCard,
+  SmallInfoCardExtraLinks,
+  StyledContentContainer,
+  StyledFarmContainer,
+  StyledHeader,
+  StyledHeaderLeft,
+  StyledHeaderRight,
+  StyledLine,
+  StyledRewardText,
+  StyledSmallInfoCardsContainer,
+} from './components';
+import { H1, P2 } from 'components/Typography';
+
+import { Web3Context } from 'contexts/Web3';
+
+interface IWithdraw {
+  farmSelected: boolean;
+}
+
+export const WithdrawInfo: React.FC<IWithdraw> = ({ farmSelected }) => {
+  const { address } = React.useContext(Web3Context);
+
+  return (
+    <>
+      {!address ? (
+        <div
+          css={`
+            align-items: center;
+            background: #000;
+            display: flex;
+            padding: ${7 * GU}px;
+          `}
+        >
+          <div
+            css={`
+              box-sizing: border-box;
```

Comment:
Hmm, this should actually be a property set at the root level of the page's `css`... Have we been using `content-box` this entire time? 😅

Reply:
Oh, yeah that actually is in fact set [here](https://github.com/pokt-network/wpokt-app/blob/milestone/withdrawal/src/index.css#L4), so I'll remove the useless line. Fixed in 71975a4375568e3aabea19bccf7b2865a5342264